Automattic/jetpack
# E2E Tests: Improve tests stability

#### Changes proposed in this Pull Request:

* Remove wait of a `warning` notice in Mailchimp flow. That wait fails the tests which reuse already used subdomain (for which Mailchimp connection is set up already)
* Explicitly wait for Jetpack dashboard to disappear after the "Connect" button is clicked. Currently, we wait ~20sec for next page to be loaded, but it is not always enough.

#### Jetpack product discussion


1156386383419466-as-1200969322108009
#### Does this pull request change what data or activity we track or use?


n/a
#### Testing instructions:






* Make sure the tests are green
* Re-run the tests, and see if they are failing
*
